content/pages/download.md

```diff
@@ -32,14 +32,7 @@ versions:
           Using the PPA ensures that new package versions will be installed automatically with `apt`. Otherwise, you can download individual packages and install them manually.
         note: |
           **Note:** Ubuntu also provides a version of Mixxx which can be installed directly from the Ubuntu Software Centre. This version is usually woefully out of date; therefore using the PPA is advised.
-        packages:
-        - slug: ubuntu32
-          name: 32-Bit
-          url: https://downloads.mixxx.org/builds/2.3/release/mixxx-2.3.0-beta-2.3-release-bionic-i386-latest.deb
-        - ubuntu64:
-          name: 64-Bit
-          url: https://downloads.mixxx.org/builds/2.3/release/mixxx-2.3.0-beta-2.3-release-bionic-amd64-latest.deb
-      - slug: fedora
+       - slug: fedora
```

Comment:
The line change breaks yaml formatting

Reply:
Fixed

Reply:
Is there a way to check the yaml format with pre-commit?